scalafix-core/src/main/scala/scalafix/rewrite/Scalameta17.scala

```diff
@@ -0,0 +1,40 @@
+// scalafmt: { maxColumn = 120, style = defaultWithAlign }
+package scalafix.rewrite
+
+import scala.collection.immutable.Seq
+import scala.meta._
+import scala.meta.semantic.v1.Completed
+import scala.meta.semantic.v1.Mirror
+import scala.meta.semantic.v1.Signature
+import scala.meta.semantic.v1.Symbol
+import scalafix.util.Patch
+import scalafix.util.TokenPatch._
+import scalafix.util.TreePatch.RenameSymbol
+
+case object Scalameta17 extends Rewrite[Mirror] {
```

Comment:
I agree! However, I still think the API is too complex. In particular, it's annoying to 

- set up test suite
- set up rewrite boilerplate (`import scalafix._`)
- copy-paste symbol syntax from mirror.database

I believe some form of boilerplate generator + simpler UI to find symbols (instead of copy-pasting from mirror.database) would go a long  way.